services/system/Accounts/plugin/src/lib.rs

```diff
@@ -29,9 +43,40 @@ impl Accounts for AccountsPlugin {
         Err(NotYetImplemented.err("add_auth_service"))
     }
 
+    fn get_account(name: String) -> Result<Option<AccountTypes::Account>, CommonTypes::Error> {
+        let acct_num = AccountNumber::from_exact(&name).map_err(|err| InvalidAccountName.err(err.to_string().as_str()))?;
+
+        let query = format!(
+            "query {{ getAccount(account: \"{}\") {{ accountNum, authService, resourceBalance }} }}",
+            acct_num 
+        );
+
+        let account_result = Server::post_graphql_get_json(&query)
+            .map_err(|e| QueryError.err(&e.message))
+            .and_then(|result| {
+                serde_json::from_str(&result).map_err(|e| QueryError.err(&e.to_string()))
+            });
+
+        let account = account_result.and_then(|response_root: ResponseRoot| {
+            response_root
+                .data
+                .getAccount
+                .ok_or_else(|| AccountDNE.err(&name))
+        })?;
```

Comment:
Sorry, just realized one more issue. This actually shouldn't throw an accountDNE error. This function returns a result<option<...>> so a valid return value is Ok(None)